validator/testdata/feature_tests/mandatory_dimensions.out

```diff
@@ -125,7 +125,11 @@ feature_tests/mandatory_dimensions.html:96:4 The tag 'amp-iframe' requires inclu
 feature_tests/mandatory_dimensions.html:100:4 The tag 'amp-fit-text' requires including the 'amp-fit-text' extension JavaScript. (see https://www.ampproject.org/docs/reference/components/amp-fit-text) [MANDATORY_AMP_TAG_MISSING_OR_INCORRECT]
 |      <amp-carousel height="42"></amp-carousel>
 >>     ^~~~~~~~~
-feature_tests/mandatory_dimensions.html:101:4 The tag 'amp-carousel' requires including the 'amp-carousel' extension JavaScript. (see https://www.ampproject.org/docs/reference/components/amp-carousel) [MANDATORY_AMP_TAG_MISSING_OR_INCORRECT]
+feature_tests/mandatory_dimensions.html:101:4 The mandatory attribute 'lightbox' is missing in tag 'amp-carousel with lightbox attribute'. (see https://www.ampproject.org/docs/reference/components/amp-carousel) [AMP_TAG_PROBLEM]
```

Comment:
This change was an unintended side effect of adding a reference point to `<amp-carousel>` for `lightbox`. @Gregable is it possible to write the reference point so that this doesn't happen? Ideally it should just say that the `amp-carousel` extension is necessary for `<amp-carousel>`.

Reply:
@honeybadgerdontcare do you have any thoughts on this?